package exception;

public class ExceptionTest {

	public static void main(String[] args) {
		int[] arr = {1,2,3,4,5};
		
		for(int i=0;i<6;i++)
			System.out.println(arr[i]);
		
		System.out.println("프로그램 종료");
	}

}
